layouts/product/pulumi-insights.html

```diff
@@ -64,48 +98,19 @@ <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
         </div>
     </section>
 
-    <section id="analytics" class="container my-28 mx-auto text-center px-6">
-        {{ with .Params.analytics }}
-            <div class="card p-6 lg:p-28 bg-white">
+    <section id="pricing" class="container my-20 mx-auto text-center px-6">
+        {{ with .Params.pricing }}
+            <div class="card p-6 lg:p-16 bg-white">
                 <h2 class="m-0 p-0">{{ .title }}</h2>
-                <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
-                <div class="max-w-4xl mx-auto my-8">{{ .description | markdownify }}</div>
-                <img class="mx-auto my-8 rounded shadow-lg" src="{{ .image }}" alt="Pulumi Insights Analytics" />
-            </div>
-        {{ end }}
-    </section>
-
-    <div class="bg-gray-100">
-        <section id="ai" class="container mt-16 mb-20 mx-auto text-center px-6 py-8 flex flex-col lg:flex-row">
-            {{ with .Params.ai }}
-                <div class="w-full lg:w-1/2 lg:content-center lg:pr-16">
-                    <h2 class="m-0 p-0">{{ .title }}</h2>
-                    <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
-                    <div class="max-w-4xl mx-auto my-8">{{ .description | markdownify }}</div>
-                    <div class="w-full flex justify-center items-center">
-                        <div class="h-12">
-                            <a class="btn-primary" href="https://app.pulumi.com">Try it Now</a>
-                        </div>
-                    </div>
-                </div>
-                <div class="w-full lg:w-1/2 content-center">
-                    <img class="mx-auto lg:mr-0 rounded shadow-lg" src="{{ .image }}" alt="Pulumi Insights AI" />
+                <p class="mx-auto">{{ .description | markdownify }}</p>
+                <div class="mt-12">
+                    <a href="/pricing" class="btn-secondary">View Pricing</a>
                 </div>
-            {{ end }}
-        </section>
-    </div>
-
-    <section id="pricing" class="container mx-auto text-center mt-28 px-6">
```

Comment:
I think pricing box needs to be the same size as the Get Started purple box below